desk/app/channels.hoon

```diff
@@ -488,11 +515,30 @@
       ca-abet:(ca-safe-sub:(ca-abed:ca-core nest) |)
     ::
         %pimp-ready
+      ?>  =(our src):bowl
       ?-  pimp
         ~         cor(pimp `&+~)
         [~ %& *]  cor
         [~ %| *]  (run-import p.u.pimp)
       ==
+    ::
+        [%sequence-numbers * @ *]
+      =+  !<([%sequence-numbers =nest:c count=@ud seqs=(list [id=id-post:c seq=@ud])] vase)
+      ?>  =(src.bowl ship.nest)
+      ?.  (~(has by v-channels) nest)  cor
+      =.  v-channels
+        %+  ~(jab by v-channels)  nest
+        |=  channel=v-channel:c
+        =.  count.channel  count
+        |-
+        ?~  seqs  channel
+        =*  next  $(seqs t.seqs)
+        ?~  p=(get:on-v-posts:c posts.channel id.i.seqs)  next
+        ?~  u.p  next
+        =.  posts.channel
+          (put:on-v-posts:c posts.channel id.i.seqs u.p(seq.u seq.i.seqs))
```

Comment:
Perhaps it is just me, but I had to look three times before understanding the precedence in the `u.p(seq.u seq.i.seq)` that lends sense to `seq.u`.